src/main/java/mate/controller/cars/CreateCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller.cars;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/create")
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/createCar.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/create.jsp").forward(req, resp);
```
it doesn't make sense to add Car prefix to the name of jsp, it's understandable, that you want to create car, taking into account the package's name, fix other jsps, where you have such case